crates/sui-indexer-alt-framework/src/cluster.rs

```diff
@@ -16,11 +15,15 @@ use tracing::info;
 use url::Url;
 
 use crate::{
-    db::DbArgs,
     ingestion::{ClientArgs, IngestionConfig},
     Indexer, IndexerArgs, IndexerMetrics, Result,
 };
 
+#[cfg(feature = "postgres")]
```

Comment:
You shouldn't need to pull these imports out and feature gate them, because the feature that enables `cluster` also implies `postgres`.